### PR TITLE
Background node-creator

### PR DESCRIPTION
**Pull request by MattiaFailla**
This PR integrates changes to the `node-creator` script by removing option parameters while also adding a `quick-start.sh` script that handles the following features:
- Download node-creator script
- Set crontab to execute node-creator on reboot
- Add `node-creator.log` file to the `$HOME` directory
- Initiate node-creator installation process

### CODE REVIEW

**Comment by Furytron** [comment]
I second this PR, it would be a fantastic addition to the library

**Comment by daveodwyer** [comment]
Hi Mett,

I've had a look through this and I am not a fan of the removal of the confirmation prompts. The idea of the prompts is that the end user knows exactly what the script is doing and is able to clearly see what they have and have not opted for. Additionally the mirror check has also been removed. This is important because the archive has gone down in the past and it offers a clear message to the end user that the archive is not going to be downloaded.

With that said, plan of action will be:

- When the script is fired it will check for an existing screen session and load it if exists
- If it does not exist, it will create it
- The script will then continue as normal

To use the AWS auto build functionality we briefly spoke about, we can add a _new_ argument that will bypass the requirement for the end user to press 'y'. This way it can be added to a build script like you suggested, but not be the default behaviour. Additionally it will mean that there is no requirement for a separate .sh file to do the build - it'll fire exactly the same.

Just a note, did you test the code that reloads the groups without a reboot? I tested that exact logic before but had issues with the groups loading in. If you've tested it, I am happy to add it in.

**Comment by MattiaFailla** [comment]
Hey Dave, 

The changes in this PR are quite different from your former plan. The current behaviour of the new install method is the following:

- Download the `node-creator` script
- Create a `crontab` entry on `reboot` to fire the `node-creator` script 
- Start the first half of the node-creator script (until the `reboot` due to user groups change)
- After startup `crontab` starts the script again and shows the logging in the home directory under the `node-creator.log` file

The user is not required to fill any parameter because `quick-start.sh` will automatically launch the script with the following paramters:
- First launch: `-dfwy`
- Second launch (reboot): `-dwy `


Edit:
These new commits revert the state of `node-creator` and force the `quick-install` script to fill parameters back in. 
Tested on `Ubuntu 20.04/Lightsail`

File: quick-start.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Download the node-creator script
+wget https://raw.githubusercontent.com/JellyPool/meter-node-creator/launch_script/node-creator
```
Comment (daveodwyer) [inline quick-start.sh]: - Fix the script github location
- Add a local location to `wget` . If the user triggers `quick-start.sh` from somewhere outside of their home directory, the crontab entry won't work.

**Pull request merged**